src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -342,6 +349,17 @@ const useResolveStep = () => {
       return;
     }
 
+    const productsWithNoRecount = cycleCountsWithItemsWithoutRecount.current
+      .map((cycleCount) => ({
+        cycleCountId: cycleCount.id,
+        product: cycleCount.cycleCountItems[0].product.productCode,
+      }));
+    // TODO: The const above just takes the cycle counts with items without recount and returns product code that should be displayed in the modal
+    // TODO: In my opinion we can hardcode QOH = 0 in the modal.
+
+    // TODO: Display the modal, pass the productsWithNoRecount. Remember we might have a few products (tables) empty, and we eventually want to call Cancel for every product
+    // TODO: So we need to do call Cancel for each "productsWithNoRecount" (this is why I store the cycleCountId also)
+
```

Comment:
here I explained what's left in terms of the modal. Please, also make sure we don't need any nullsafes around the `cycleCountItems[0]` (probably not, but just to be sure)

Reply:
thanks for doing that! It was very helpful for me :thumbsup: